spring-petclinic-microservices/spring-petclinic-config-server/pom.xml

```diff
@@ -3,16 +3,17 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
-	<groupId>org.springframework.samples.petclinic.config</groupId>
-	<artifactId>spring-petclinic-config-server</artifactId>
-	<packaging>jar</packaging>
-	<description>Spring PetClinic Config Server</description>
+    <groupId>org.springframework.samples.petclinic.config</groupId>
```

Comment:
nit: How about uniform the groupId? `com.azure.spring`, and it can be deleted because it's same to parent.

Reply:
the petclinic has many work to do,
change the groupId is not in this pr's scope,
all the dependencies, names and etc is tracked in this [issue](https://github.com/Azure/azure-sdk-for-java/issues/26234).

Reply:
I suggest we don't change petclinic group id